### ISSUE TITLE
jtaglib support for the MSP430Xv2 architure

### ISSUE THREAD

**Issue by smunaut** [dlbeer/mspdebug#87]
AFAICT all the commands in jtaglib (and even the JTAG ID matching) are for the classic MSP430 architecture and there is no support for the FRAM devices or any MSP430Xv2 architecture devices.

**Comment by dlbeer** [comment]
Yes, that's probably true. If anyone wants to contribute a set of patches to improve jtaglib or the JTAG -based drivers I'd be happy to integrate them.
